phpstan-baseline.neon

```
@@ -92250,3 +92250,7 @@ parameters:
 			count: 1
 			path: plugins/MauticTagManagerBundle/Tests/Unit/Security/Permissions/TagManagerPermissionsTest.php
 
+		-
```

@escopecz this error was ignored for other plugins so I ignored it as well

---

I'm OK with this since the goal of this PR is to remove the Citrix plugin altogether. Thanks for pointing this out!